CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 <!-- markdownlint-disable MD041 -->
+## 0.23.0 (Unreleased)
```

Comment:
@tamirkamara wanting to release 0.22 before this is merged?

Reply:
@marrobi unsure what you're asking but I don't have an opinion which version should include this feature.

Reply:
Ok, thought it was under 0.23.0 as a result of a comment. Will move under 0.22.0 then.